package.json

```diff
@@ -8,10 +8,12 @@
     "clean": "rimraf dist",
     "start": "webpack-dev-server --hot --inline --progress",
     "build": "webpack --progress",
-    "prod": "npm run clean && webpack -p --progress"
+    "prod": "npm run clean && webpack -p --progress",
+    "winstop": "Taskkill /IM node.exe /F"
```

Comment:
As far as I develop on a windows OS, I added this script to help to kill all the node running process that still run after `ctrl + c`  usage command in the console